packages/preset-create-react-app/helpers/checkPresets.js

```diff
@@ -0,0 +1,17 @@
+const { logger } = require('@storybook/node-logger');
+
+const incompatiblePresets = ['@storybook/preset-scss', '@storybook/preset-typescript'];
+
+const checkPresets = () =>
+  incompatiblePresets.forEach(preset => {
+    try {
+      require.resolve(preset);
+      logger.warn(
+        `\`${preset}\` may not be compatible with \`@storybook/preset-create-react-app\``
```

Comment:
Beautiful 😍

Reply:
This checks if the presets are installed. Slightly more accurate if we check if they're used in the user's `presets.js` file? Not a big deal.

Reply:
Will do.